src/pydata_sphinx_theme/assets/scripts/bootstrap.js

```diff
@@ -0,0 +1,16 @@
+/* Sphinx injects the html output with jquery and other javascript files.
+ * To enable Popper.js (and other jQuery plugins) to hook into the same instance of jQuery,
+ * jQuery is defined as a Webpack external, thus this import uses the externally defined jquery dependency.
+ */
+import "jquery";
+import "popper.js";
+
+import "bootstrap";
+
+import "../styles/bootstrap.scss";
+
+/*******************************************************************************
+ * Finalize
```

Comment:
Could you give something more descriptive than `finalize`?

Reply:
I simply copy/pasted what was there in the `index.js`. I did something more explicit for both files